src/main/scala/com/danielasfregola/twitter4s/http/clients/rest/search/TwitterSearchClient.scala

```diff
@@ -74,9 +75,13 @@ trait TwitterSearchClient {
                   until: Option[LocalDate] = None,
                   since_id: Option[Long] = None,
                   max_id: Option[Long] = None,
-                  callback: Option[String] = None): Future[RatedData[StatusSearch]] = {
+                  callback: Option[String] = None,
+                  tweet_mode: Option[String] = Some("extended")): Future[RatedData[StatusSearch]] = {
```

Comment:
`tweet_mode` should be an optional enum - see for example what we do for Language. 
Having an enum rather than a string makes the usage of the API easier, as the consumer of the API as a finite set of values to chose from - in this case, he'll choose from either no value of the value extended.

Reply:
Also, tweet_mode should be added to all the REST endpoints that support it - not just the `searchTweet` function. Happy to split this in more PRs/contributions if needed.

Reply:
Also, we need to change the default of `tweet_mode` to `None` for BC